mavlink/definitions/uavionix.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version='1.0'?>
+<mavlink>
+    <include>common.xml</include>
+    <enums>
+      <enum name="UAVIONIX_SDA_LEVEL">
```

Comment:
Since this is part of the uAvionix dialect, would there be a problem dropping the `UAVONIX_` prefix?